vm-virtio/src/device.rs

```diff
@@ -13,6 +13,8 @@ use std::sync::Arc;
 use vm_memory::GuestMemoryMmap;
 use vmm_sys_util::EventFd;
 
+pub type IrqClosure = Arc<Box<Fn(u16) + Send + Sync>>;
```

Comment:
I think the closure should be more generic to support non PCI transports. Ideally we would pass a `Queue` clone or reference to it.

Reply:
Done